diff --git a/node_modules/react-beautiful-dnd-cjs-patch/dist/react-beautiful-dnd.cjs.js b/node_modules/react-beautiful-dnd-cjs-patch/dist/react-beautiful-dnd.cjs.js
index c870603..b853549 100644
--- a/node_modules/react-beautiful-dnd-cjs-patch/dist/react-beautiful-dnd.cjs.js
+++ b/node_modules/react-beautiful-dnd-cjs-patch/dist/react-beautiful-dnd.cjs.js
@@ -5330,7 +5330,7 @@
     var _this$props = this.props,
         children = _this$props.children,
         droppableId = _this$props.droppableId;
-    var isEnabled = this.state.isEnabled;
+    var isEnabled = true;
     var Inner = this.getInner();
 
     if (!isEnabled) {
